context/app/routes_main.py

```diff
@@ -58,9 +58,9 @@ def search():
     )
 
 
-@blueprint.route('/test-search')
-def test_search():
-    title = 'Test Search'
+@blueprint.route('/test-search/<type>')
+def test_search(type):
+    title = f'{type.capitalize()} Test Search'
```

Comment:
In the existing search, the type is a query parameter, but they we jump through hoops so it doesn't actually show up for the user -- It looks like you've found a different way to do it? That sounds good!

Reply:
... but if someone tries `/test-search/not-a-type`, they'll get a soft 404 because the JS routing will fail? Soft 404s aren't great... but not the end of the world, either.